tests/flow/test_graph_copy.py

```
@@ -17,16 +19,18 @@ def graph_copy(self, src, dest):
+        # limit retries to 20
+        for _ in range(max_iterations):
             try:
                 self.env.assertTrue(graph_eq(A, B))
-                break
-            except Exception as e:
-                # retry if query was issued while redis is loading
-                if str(e) == "Redis is loading the dataset in memory":
-                    print("Retry!")
-                    continue
+                return
+            except BusyLoadingError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)